0.9.0/_static/pygments.css

```diff
@@ -1,7 +1,7 @@
 pre { line-height: 125%; }
-td.linenos pre { color: #000000; background-color: #f0f0f0; padding-left: 5px; padding-right: 5px; }
```

Comment:
same with this file -- is this okay or should it be changed back

Reply:
Can you enable your gh-pages build and skim through the doc to see if there is any obvious layout change? If not we can keep the change.